src/stuf/admin.py

```diff
@@ -70,3 +69,10 @@ class SoapServiceAdmin(PrivateMediaMixin, admin.ModelAdmin):
             },
         ),
     )
+
+
+@admin.register(SoapService)
+class SoapServiceAdmin(admin.ModelAdmin):
+    class Meta:
+        model = SoapService
+        fields = "__all__"
```

Comment:
This looks like `ModelForm` things? but that's not what the django admin uses.

Please make sure to explicitly specify `list_display` for the admin (and `search_fields`)